test/Fondue.test.js

```diff
@@ -104,6 +104,18 @@ describe("isColor", () => {
 	});
 });
 
+describe("hasColorTable", () => {
+	test("color font", async () => {
+		const fondue = await colorFont();
+		expect(fondue.colorFormats).toContain("COLR");
+	});
+
+	test("non color font", async () => {
+		const fondue = await ttfFont();
+		expect(fondue.colorFormats).toStrictEqual([]);
```

Comment:
I think it's fine to return an empty list here. Returning null would be a bit unexpected IMO.

Reply:
👍